oop.md

```diff
@@ -165,20 +165,17 @@ Output:
 
 **How It Works**
 
-To use inheritance, we specify the base class names in a tuple following the class name in the class definition. Next, we observe that the `__init__` method of the base class is explicitly called using the `self` variable so that we can initialize the base class part of the object. This is very important to remember - Since we are defining a `__init__` method in `Teacher` and `Student` subclasses, Python does not automatically call the constructor of the base class `SchoolMember`, you have to explicitly call it yourself.
+To use inheritance, we specify the base class names in a tuple following the class name in the class definition (ex. `class Teacher(SchoolMember)`).   Next, we observe that the `__init__` method of the base class is explicitly called using the  `self`  variable so that we can initialize the base class part of an instance in the subclass. This is very important to remember- Since we are defining a  `__init__`  method in `Teacher`  and  `Student`  subclasses, Python does not automatically call the constructor of the base class  `SchoolMember`, you have to explicitly call it yourself.
```

Comment:
why `ex.`? I haven't seen that notation before.

Reply:
Oh, I meant it as an abbreviation for "example".  Some people use "e.g." instead and would argue that "e.g." is the way to go.

Reply:
Let's just use verbose `for example`, to keep it simple.